docs/get_started.ipynb

```diff
@@ -88,7 +84,7 @@
     },
     {
       "cell_type": "code",
-      "execution_count": 0,
+      "execution_count": null,
```

Comment:
Does this have any effect on the doc as it gets uploaded to the website?

Reply:
No idea, this is the result of resetting the notebook and running nbfmt.  I suspect they are equivalent in notebooks that are in a 'not-run-yet' state.